tests/misc_test.go

```diff
@@ -168,7 +168,7 @@ func validateSspIsFailingToReconcileMetric() {
 var _ = Describe("SCC annotation", func() {
 	const (
 		sccAnnotation = "openshift.io/scc"
-		sccRestricted = "restricted"
+		sccRestricted = "restricted*"
```

Comment:
This is probably a typo. It would correctly match `restricted-v2`, but also other strings like: `prefix-resticte-suffix`.
Can you change it to `^restricted` ?